src/components/ProfileCard.css

```diff
@@ -0,0 +1,33 @@
+.CreatedProfile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 1em;
+}
+
+.ProfilePictureWrapper {
+  background-color: whitesmoke;
+}
+
+.ProfileHeading {
+  display: flex;
+  background-color: whitesmoke;
+  padding: 25px;
+  color: thistle;
+  border-radius: 15px;
+  letter-spacing: 4px;
+}
+
+.ProfileCardInput {
+  margin-top: 2em;
+  background-color: thistle;
+  padding: 25px;
+  color: white;
+  border-radius: 40px;
+}
+
+.Htag {
```

Comment:
This is not a recommended class name. The class name should represent what the h4 is used for.  If you want to style all `h4`  just use `h4` as a selector. If you want to style the `h4` inside the `CreatedProfile` element, then do a `.CreatedProfile h4` selector.